db/schema.rb

```diff
@@ -45,7 +58,9 @@
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "item_id", null: false
     t.string "seller_gift_card_id"
+    t.bigint "recipient_id"
```

Comment:
This should be nonnullable

Reply:
cool, I'll add in that to the migration and make the model change as well